steps/sms-notify/enter-mobile/EnterMobile.js

```diff
@@ -14,28 +14,25 @@ class EnterMobile extends Question {
         return 'sms-notify/enter-mobile/template';
     }
 
-    get i18NextContent() {
-        return content;
-    }
-
     get form() {
 
-        const fields = this.content.fields;
-        const validation = [{
-            regex: whitelist,
-            msg: 'emptyField'
-        }, {
-            regex: mobileNumber,
-            msg: 'invalidNumber'
-        }];
-
         return form(
             field('mobileNumber')
-                .validate(multiRegex(validation, fields.mobileNumber))
-                .content(fields.mobileNumber)
+                .joi(
+                    this.content.fields.mobileNumber.error.emptyField,
+                    Joi.string().required()
+                )
+                .joi(
+                    this.content.fields.mobileNumber.error.invalidNumber,
+                    Joi.string().regex(mobileNumber).required()
```

Comment:
As the validators are run in sequence I think you _should_ be able to drop the `.string()` and `.required()` since this one will only run if the one above passed. At least that's what I think should happen.

Reply:
I take your point, I'm not sure how would I define the second schema by dropping both `.string()` and `.required()`?

Reply:
Yep, my mistake. Hadn't realised how joi structures it's schema.